src/components/common/Breadcrumbs.tsx

```diff
@@ -28,7 +28,7 @@ const Breadcrumbs = (props: BreadcrumbsPropsDefault) => {
   const { relativePrefixToRoot } = useTemplateData();
 
   return (
-    <>
+    <ErrorBoundaryWithAnalytics name="breadcrumbs">
```

Comment:
Is there any specific reason why this was in the template file instead of in each component file?

Reply:
There wasn't a specific reason and I agree that it declutters things by moving it into the component files. However if you're going to do this then we should also do it for all the other [components](https://github.com/yextconsulting/pages-starter-react-consulting/blob/a760ee2b75cea0e26da2f4e780a06142f144fbc8/src/layouts/entity.tsx) for consistency